Documentation/git-config.txt

```diff
@@ -235,8 +235,12 @@ FILES
 If not set explicitly with `--file`, there are four files where
 'git config' will search for configuration options:
 
+$PROGRAMDATA/Git/config::
+	System-wide configuration file for *all* Git-related software running
```

Comment:
If you read on you will find a part where it is described that `--system` will target the _system-wide configuration_. If you leave the new text as-is, how does a user know which one `--system` targets?